plugins/labhub.py

```diff
@@ -302,11 +302,27 @@ def difficulty_level(user, iss):
                 elif self.GH3_ORG.is_member(user):
                     return True
 
-        def eligible(user, iss):
-            for chk in checks:
-                if not chk(user, iss):
+            @register_check
+            def block_gci_issue_assignment(user, iss):
+                """
+                True if the issue is not labelled with 'initiatives/gci'.
+                False if the issue has been labelled with 'initiatives.gci'.
+                """
+                if 'initiatives/gci' in iss.labels:
                     return False
-            return True
+                else:
+                    return True
+
+        def eligible(user, iss):
+            """
+            True if all checks are passed.
+            False if any check(s) is/are failed.
+            """
+            el = True
+            for check in checks:
+                # el evaluates to False if any check is failed.
+                el = el and check(user, iss)
```

Comment:
why are you changing this method? The other one was more efficient because it didn't run checks if a previous one already failed (that'd be redundand right? `False and <whatever>` will always be false)